packages/remark-mdx/index.js

```diff
@@ -1,7 +1,7 @@
 const IMPORT_REGEX = /^import/
 const EXPORT_REGEX = /^export/
 const EXPORT_DEFAULT_REGEX = /^export default/
-// const BLOCKS_REGEX = '[a-z\\.]+(\\.){0,1}[a-z\\.]'
+// Const BLOCKS_REGEX = '[a-z\\.]+(\\.){0,1}[a-z\\.]'
```

Comment:
It was automatically rewritten, but is this a good change?

Reply:
I think you can just remove the whole comment. It's in git if we ever need it!

Reply:
Oops, I didn't think about this. Yeah, remove it! 🔥